### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by pdx-cs-lbryan**
This seems to be functioning in the current context by running app.py and going to {server address}/form/review, though I'm not sure how well it currently matches the MVC model (feedback there would be very much appreciated)

There's basic error handling, and I intend on cleaning this up as well as adding formal tests. I also need to put in some styling.

I'm still waiting on some changes to capstone360.db to use it for test data, so for now I'm using the test db/entries I implemented (there's some code in app.py that will be removed when the database is updated)

Any coding/style improvement suggestions would be welcome, but the priority is currently anything that looks broken.

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    #from . import db
+    db.init_app(app)
+    with app.app_context():
+        db.drop_all()
+        db.create_all()
+        fill_data(db)
+
+    from . import form
+    app.register_blueprint(form.bp)
```
Comment (J8cob1) [inline __init__.py:100]: May I ask you to explain what the blueprint is for?

**Review by J8cob1 — CHANGES_REQUESTED** [review]
So something I noticed is that db_form.py and form.py are in two different places. Is this necessary?

File: __init__.py (hunk 1 above)
Comment (pdx-cs-lbryan) [reply __init__.py:100]: It's not required, but I'm using it to hold together the various pieces of forms. To some extent it made more sense to have a blueprint when I had midterms and finals as their own pages. Currently it's just holding the review page and the submission page. 

If you're unfamiliar with blueprints, the idea is that they're components/subdivisions of a project. Their chief purpose is to help organize a project and localize those things that apply to certain contexts. See: http://flask.pocoo.org/docs/1.0/blueprints/

**Comment by pdx-cs-lbryan** [comment]
> So something I noticed is that db_form.py and form.py are in two different places. Is this necessary?

db_form.py is the file that sets up the database. It might be changed to link to a pre-existing database like Faisal currently has with your code, or that might be taken care of elsewhere to work with the MVC format (in which case db_form would be deleted).

form.py is the file that holds the functions for actually setting up and processing the reviews.

**Pull request closed without merging**